Dockerfile

```diff
@@ -38,7 +38,12 @@ RUN \
     #Debian: useradd --create-home --user-group $USER --shell /bin/bash && \
     adduser -D -h /app -s /bin/sh $USER && \
     # Allow user to access the build
-    chown -R $USER.$USER /app
+    chown -R $USER:$USER /app
+
+# Use of Microsoft.Data.SqlClient requires ICU to be installed
+# https://github.com/dotnet/dotnet-docker/blob/main/samples/enable-globalization.md
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false
+RUN apk add --no-cache icu-libs
```

Comment:
Could you move the env down after the other envs?
Also, merge the apk command with the RUN above? (to avoid extra layers)

Reply:
Changes have been pushed. I actually started with the command in the existing RUN but moved them out to keep the two settings together with the link. I've added a relevant comment to each line instead.